etc/mediathekview.profile

```diff
@@ -23,6 +28,7 @@ protocol unix,inet,inet6
 seccomp
 tracelog
 
+disable-mnt
```

Comment:
This can save/playback video files yea? Won't some people have that all on another drive?